Manuals and specifications (#3)
javascript-tutorial/nl.javascript.info: 1-js/01-getting-started/2-manuals-specifications/article.md

# Handleidingen en specificaties

> Dit boek is een *tutorial*. Het is bedoeld om u te helpen de taal geleidelijk aan te leren. Maar als je eenmaal vertrouwd bent met de basis, heb je andere bronnen nodig.

Reviewer: ```suggestion
Dit boek is een *tutorial*. Het is bedoeld om je te helpen de taal geleidelijk aan te leren. Maar als je eenmaal vertrouwd bent met de basis, heb je andere bronnen nodig.
```

## Specificatie

De ECMA-262-specificatie (https://www.ecma-international.org/publications/standards/Ecma-262.htm) bevat de meest diepgaande, gedetailleerde en geformaliseerde informatie over JavaScript. Het definieert de taal.

Maar omdat het zo geformaliseerd is, is het in eerste instantie moeilijk te begrijpen. Dus als u de meest betrouwbare bron van informatie over de taaldetails nodig hebt, is de specificatie de juiste plaats. Maar het is niet voor dagelijks gebruik.

Elk jaar komt er een nieuwe versie van de specificatie uit. Tussen deze releases door staat de laatste specificatie draft op <https://tc39.es/ecma262/>.

Om te lezen over goednieuwe functies, inclusief diegene "bijna standaard" zijn (de zogenaamde "fase 3"), zie de voorstellen op <https://github.com/tc39/proposals>.

> Ook, als u in ontwikkeling bent voor de browser, dan zijn er andere specificaties behandeld in het [tweede deel](info:browser-omgeving) van de tutorial.

Reviewer: ```suggestion
Voor ontwikkelaars van browserscript/-applicaties worden meer daarop gerichte specificaties behandeld in het [tweede deel](info:browser-omgeving) van de tutorial.
```

## Handleidingen

- **MDN (Mozilla) JavaScript Referentie** is een handleiding met voorbeelden en andere informatie. Het is geweldig om diepgaande informatie te krijgen over individuele taalfuncties, methoden, enz.

    Men kan deze vinden op <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference>.

    Hoewel het vaak het beste is om een internet zoekopdracht te gebruiken. Gebruik gewoon "MDN [term]" in de query, bijvoorbeeld <https://google.com/search?q=MDN+parseInt> om te zoeken naar de `parseInt` functie.

- **MSDN** - Microsoft handleiding met veel informatie, waaronder JavaScript (vaak JScript genoemd). Als men iets specifieks nodig heeft voor Internet Explorer, kan men beter de volgende website gebruiken: <http://msdn.microsoft.com/>.

    Ook kunnen we gebruik maken van een internet zoekopdracht met zinnen als "RegExp MSDN" of "RegExp MSDN jscript".

## Compatibiliteitstabellen

> JavaScript is een ontwikkelende taal, er worden regelmatig nieuwe functies toegevoegd.

Reviewer: ```suggestion
JavaScript is een taal die voortdurend in ontwikkeling is. Er worden regelmatig nieuwe functies toegevoegd.
```

> Om hun ondersteuning te zien onder browser-gebaseerde en andere engines, zie:

Reviewer: ```suggestion
Om te zien welk javascript browser- en andere engines momenteel ondersteunen, zie:
```

- <http://caniuse.com> - ondersteuningstabellen per functie, bijvoorbeeld om te zien welke engines moderne cryptografische functies ondersteunen: <http://caniuse.com/#feat=cryptography>.
- <https://kangax.github.io/compat-table> - een tabel met taalfuncties en engines die deze wel of niet ondersteunen.

Al deze bronnen zijn nuttig in de praktijk, omdat ze waardevolle informatie bevatten over taaldetails, hun ondersteuning, enz.

> Onthoud ze (of deze pagina) voor de gevallen waarin u diepgaande informatie over een bepaalde functie nodig hebt.

Reviewer: ```suggestion
Onthoud ze (of deze pagina) voor de gevallen waarin u diepgaande informatie over een bepaalde functie nodig heeft.
```
